projects.json

```diff
@@ -4782,7 +4789,23 @@
     "actions": [
       {
         "action": "BuildSwiftPackage",
-        "configuration": "release"
+        "configuration": "release",
+        "xfail": [
+          {
+            "issue": "rdar://124167314",
+            "compatibility": ["5.9"],
+            "branch": ["release/5.9"],
+            "job": ["source-compat"],
+            "platform": "Darwin"
+          },
+          {
+            "issue": "rdar://124167314",
+            "compatibility": "5.9",
+            "branch": ["main", "release/5.9", "release/5.10"],
```

Comment:
Then this can be `"main", "release/5.10"` with some like "sqlite3 missing on Linux". Not sure what to do there, @shahmishal are we able to install sqlite3 on the Linux image used for this job?